tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Why do we have this file now?

Reply:
When running Eslint running from the root, it reads also TS files, which requires a tsconfig in the root.